recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: e4afaf5a1383c1a3be00fd4870877a1b1b6e4f893823318146a60138cbc38b38
 
 build:
-  number: 1
+  number: 2
```

Comment:
The current latest version is 2022.07.0 with the build number 0. This PR will not change the version but the build number, so its build number needs to be incremented to 1, not to 2

Reply:
fixed.